docs/en/observability/ingest-logs.asciidoc

```diff
@@ -0,0 +1,188 @@
+[chapter, role="xpack"]
+[[observability-ingest-logs]]
+
+:beatname_uc: Filebeat
+:beatname_lc: filebeat
+:beatname_url: {beats-ref-root}/{beatname_lc}/{branch}
+
+[[ingest-logs]]
+= Ingest logs
+
+{beatname_uc} allows you to read, preprocess, and ship data from sources that come
+in the form of files. {filebeat} monitors the log files or locations that you specify,
+collects log events, and forwards them to {es}. To ease the collection and parsing of
+log formats for common applications such as Apache, MySQL, and Kafka, a number of
+{filebeat-ref}/filebeat-modules.html[modules], also known as
+https://www.elastic.co/integrations?solution=observability[integrations], are available.
+
+[NOTE]
+=====
+If you haven't already, you need to install {es} for storing and searching your data, and {kib} for visualizing and
+managing it. For more information, see <<install-observability,Get started>>.
+=====
+
+[[install-filebeat]]
+== Step 1: Install {beatname_uc}
+
+Install {beatname_uc} on all the servers you want to monitor.
+
+To download and install {beatname_uc}, use the commands that work with your system:
+
+include::{beats-repo-dir}/tab-widgets/install-widget.asciidoc[]
+
+[[other-filebeat-installations]]
+=== Other installation options
+
+* {filebeat-ref}/running-on-kubernetes.html[Kubernetes]
+* {filebeat-ref}/running-on-docker.html[Docker]
+* {filebeat-ref}/running-on-cloudfoundry.html[Cloud Foundry]
+* {filebeat-ref}/setup-repositories.html[APT or YUM]
+* https://www.elastic.co/downloads/beats/filebeat[Download page]
+
+[[logs-connect-to-stack]]
+== Step 2: Connect to {es} and {kib}
+
+Connections to {es} and {kib} are required to set up Filebeat.
+
+Set the connection information in `filebeat.yml`. To locate this configuration file,
+see {filebeat-ref}/directory-layout.html[Directory layout].
+
+include::{beats-repo-dir}/tab-widgets/set-connection-widget.asciidoc[]
```

Comment:
Hmmm... I wonder if we should add a tip to the widget that tells users how where to find the cloud ID? (The link is OK but sends users away from the main flow.)

Reply:
For future improvements, I created an issue for this: #78